templates/wizard.php

```diff
@@ -59,7 +59,7 @@
 	and <a href="http://owncloud.org/promote"
 		target="_blank">spread the word</a>!')); ?>
 <?php else: ?>
-© 2011-<?php p(date('Y')) ?> <a href="https://owncloud.com" target="_blank">ownCloud Inc.</a>
+© <?php p(date('Y')) ?> <a href="<?php print_unescaped($theme->getBaseUrl()); ?>" target="_blank"><?php print_unescaped($theme->getEntity()); ?></a>
```

Comment:
In the a tag use p() instead of print_unescaped(). p() escapes html and as a result prevents injecting html into the href attribute. Actually also use it for the entity. Unless you have seen cases where it contains HTML.